x/liquidstaking/types/params_test.go

```diff
@@ -12,7 +12,7 @@ func TestParams(t *testing.T) {
 	require.IsType(t, paramstypes.KeyTable{}, ParamKeyTable())
 
 	params := DefaultParams()
-	paramStr := `dynamic_fee_rate:<r0:"0" u_soft_cap:"50000000000000000" u_hard_cap:"100000000000000000" u_optimal:"90000000000000000" slope1:"100000000000000000" slope2:"400000000000000000" max_fee_rate:"500000000000000000" > `
+	paramStr := `dynamic_fee_rate:<r0:"0" u_soft_cap:"50000000000000000" u_hard_cap:"100000000000000000" u_optimal:"90000000000000000" slope1:"100000000000000000" slope2:"400000000000000000" max_fee_rate:"500000000000000000" > maximum_discount_rate:"30000000000000000" `
```

Comment:
These are `sdk.dec` types, but I think they are printed incorrectly, we should correct them by using a separate string function, not the string of the proto

add `option (gogoproto.goproto_stringer) = false;` on .proto

and add string function on types/params.go
```
// String returns a human-readable string representation of the parameters.
func (p Params) String() string {
	out, _ := yaml.Marshal(p)
	return string(out)
}
```

Reply:
Thanks for the tip! I'll add custom String method :) Thanks.

Reply:
done! https://github.com/b-harvest/Canto/pull/18/commits/b00c46802ee670ff14f0069d6fd830b0dddd8d2a